recipes/_install-windows.rb

```diff
@@ -40,8 +40,11 @@
   dd_agent_latest = 'ddagent-cli-latest'
 end
 
+# If a different file prefix is specified, use that:
+dd_agent_installer_prefix = node['datadog']['windows_installer_prefix'] ? node['datadog']['windows_installer_prefix'] : 'ddagent-cli'
```

Comment:
The attribute `windows_installer_prefix` must be added to `attributes/default.rb`, and documented there.

Also, nit: for consistency with other Windows-related attributes, I slightly prefer a name such as `windows_agent_installer_prefix`, even though it's longer